scripts/funcTests/runFuncTests.sh

```diff
@@ -12,6 +12,10 @@ done
 
 RESULTCODE=0
 
+# check openssl version
```

Comment:
```suggestion
# print openssl version
```

check implies a validation, and there's none :)

Reply:
Make sense. Fixed. Thanks!